defaults/main.yml

```diff
@@ -1,2 +1,34 @@
+# Defaults file for package_updater
+
+# List of packages to update, (default) install, or remove
+package_updater_packages: []
+
+# Examples:
```

Comment:
In general, our levels of documentation like this belong in README.md. Documentation of why and how goes in the code, what belongs in the user-facing docs.

Reply:
removed